DigitalExcellence/dex-frontend
# Feature/505 add my projects page

## Description

A few sentences describing the overall goals of the pull request's commits.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Login with identity server or FHICT account
2. If the logged in user hasn't uploaded any projects, upload a project
3. Hover over your profile picture (top right corner)
4. Click on my projects
5. See (all) your project(s) appear here

https://github.com/DigitalExcellence/dex-frontend/issues/505

Closes: #505
